certdepot/impl.go

```diff
@@ -0,0 +1,180 @@
+package certdepot
+
+import (
+	"time"
+
+	"github.com/mongodb/anser/bsonutil"
+	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/message"
+	"github.com/pkg/errors"
+	"github.com/square/certstrap/depot"
+	mgo "gopkg.in/mgo.v2"
+)
+
+type DBUser struct {
```

Comment:
can we call this just User? 

this is just the same model as the other user collection right?

Reply:
No it is a different kind of user

Reply:
but it's the same model?